editor/test/resources/test_project/mesh/DamagedHelmet.gltf

```diff
@@ -0,0 +1,204 @@
+{
```

Comment:
Test data is from Khronos:
https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/DamagedHelmet

Reply:
We should include the license information since it is Attribution-licensed.